apps/myaccount/src/components/login-verify-data/login-verify-data.tsx

```diff
@@ -55,11 +55,17 @@ export const LoginVerifyingData: React.FunctionComponent<LoginVerifyDataProps> =
     const [typingDNAEnabled, setTypingDNAEnabled] = useState(false);
 
     const getTypingDNAEnabled = (): void => {
-        isTypingDNAEnabled().then(function(data) {
-            setTypingDNAEnabled(data);
-        }).catch(() => {
-            setTypingDNAEnabled(false);
-        });
+        if (isFeatureEnabled(
+            featureConfig?.security,
+            AppConstants.FEATURE_DICTIONARY.get("SECURITY_LOGIN_VERIFY_DATA_TYPINGDNA")
+        )) {
+            isTypingDNAEnabled().then(function(data) {
+                setTypingDNAEnabled(data);
+            }).catch(() => {
+                setTypingDNAEnabled(false);
+            });
+        }
+        setTypingDNAEnabled(false);
```

Comment:
If this is not included in a ‘else’ clause, the feature cannot be enabled even when the config is enabled from the config file.